src/Components/Search-Bar.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import Option from './Search-Options'
+import {currentSearchString, resetSelection} from '../Actions/searchActions'
+import {connect} from 'react-redux'
+
+// var options = [{id: 1, name:'this'},{id: 2, name:'this'}, {id: 3, name:'this'}]
+const SearchBar = (props) => {
+    return ( 
+        <div className="search-container">
+            <input 
+            className="Search-Bar"
+            id="Search-Bar-Id"
+            type="search" 
+            placeholder="Search Products Here"
```

Comment:
The default ebay placeholder here is "Search for anything", please update this before you merge!